src/test/resources/com/jpinpoint/perf/lang/java/ruleset/common/xml/BufferFilesNewStream.xml

```diff
@@ -74,4 +74,34 @@ class Issue273 {
      ]]></code>
     </test-code>
 
+    <test-code>
+        <description>violation: BufferFilesNewStream Issue #456</description>
+        <expected-problems>1</expected-problems>
+        <expected-linenumbers>7</expected-linenumbers>
+        <code><![CDATA[
+import java.io.*;
+import java.nio.file.*;
+import org.apache.commons.io.IOUtils;
+
+class Issue456 {
+    void ioUtilsBad(InputStream in, String path) throws IOException {
+        OutputStream out = Files.newOutputStream(Paths.get(path)); // bad, not buffered
+        IOUtils.copy(in, out);
```

Comment:
what if out is not File based? E.g. network or in memory?

Reply:
It is about the Files class, so meant for files not other I/O.

Reply:
IOUtils is for other I/O than just files, yet Files.newXXX is meant for files (and directories)